xbmc/pvr/PVRDatabase.cpp

```diff
@@ -45,7 +45,7 @@ void CPVRDatabase::CreateTables()
 {
   CLog::Log(LOGINFO, "PVR - %s - creating tables", __FUNCTION__);
 
-  CLog::Log(LOGDEBUG, "PVR - %s - creating table 'channels'", __FUNCTION__);
+  CLog::Log(LOGDEBUG, "CPVRDatabase - %s - creating table 'channels'", __FUNCTION__);
```

Comment:
Its currently not unique at all, many place in kodi are logged with __Classname__, and also in PVR Section there is a mix of Classname output and simply "PVR". Because I spend the most of my time with searching the correct files using rgrep, I think it's not so bad to get an hint directly from log output. I left the Notices as they are so normal users see what they have seen before.